packages/orchestra/lib/app.ts

```diff
@@ -0,0 +1,32 @@
+import './tracer.js';
+import { getLogger } from '@nangohq/utils';
+import { getServer } from './server.js';
+import { envs } from './env.js';
+import { migrate, Scheduler } from '@nangohq/scheduler';
+
+const logger = getLogger('Orchestra');
+
+try {
+    await migrate();
+
+    // TODO: add logic to update syncs and syncs jobs in the database
+    const scheduler = new Scheduler({
+        on: {
+            CREATED: (task) => console.log(`Task ${task.id} created`),
```

Comment:
use logger

Reply:
This is a big TODO, the console.logs will go away soon